### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by k2helix**

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <string name="ok">OK</string>
+    <string name="cancel">Cancelar</string>
+    <string name="confirm">Confirmar</string>
+    <string name="are_you_sure_delete_medicine">¿Quieres eliminar el medicamento?</string>
+    <string name="no_reminders">Ningún recordatorio establecido</string>
+    <plurals name="sum_reminders">
+        <item quantity="one">%1$d recordatorio\n%2$s</item>
+        <item quantity="other">%1$d recordatorios\n%2$s</item>
+    </plurals>
+    <string name="channel_name" translatable="false">MedTimer</string>
```
Comment (Futsch1) [inline app/src/main/res/values-es/strings.xml]: Please remove the translations with the flag "translatable" set to false from the XML.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <string name="channel_description" translatable="false">Notificaciones de MedTimer</string>
```
Comment (Futsch1) [inline app/src/main/res/values-es/strings.xml]: Please remove the translations with the flag "translatable" set to false from the XML.

Comment (k2helix) [reply app/src/main/res/values-es/strings.xml]: You're right! I didn't notice them as I didn't use Android Studio to edit the file. Sorry!

**Comment by Futsch1** [comment]
Thanks a lot for the PR, mucb appreciated!

**Comment by k2helix** [comment]
No problem! Thank you for the app!

**Comment by k2helix** [comment]
Hi @Futsch1, I noticed a typo in the medicine tab. I translated it as "Medicamento" which is a single medicine. The correct translation would be either "Medicamentos" (in plural) or "Medicación" (which means medication). Considering that the first one would require more space, I recommend the second one.

(The literal translation of medicine is "Medicina", and even though one of its meanings is the same, it's more used to refer to the profession/treatment, so it'd be confusing)

**Comment by Futsch1** [comment]
Thanks for the hint, I'll fix it

**Pull request merged**